### ISSUE TITLE
Broken sourcing from ~/.zshrc.d

### ISSUE THREAD

**Issue by Lenart12** [unixorn/zsh-quickstart-kit#101]
.zshrc does not source dotfiles correctly from ~/.zshrc as it doesnt find actual dotfiles.
Reccomended fix (made in #102): 
```zsh
mkdir -p ~/.zshrc.d
# ..... Changes
if [ -n "$(/bin/ls -A ~/.zshrc.d)" ]; then
  for dotfile in $(find ~/.zshrc.d -type f)
# .....
  do
    if [ -r "${dotfile}" ]; then
      source "${dotfile}"
    fi
  done
fi
```
Use -A swich with ls to list dotfiles and find to enumerate all files (including dotfiles)

**Comment by unixorn** [comment]
Closed by #102

**Comment by unixorn** [comment]
Thanks for the catch and thanks for submitting a fix.
